packages/actor-rdf-resolve-quad-pattern-tree/lib/ActorRdfResolveQuadPatternTree.ts

```diff
@@ -0,0 +1,147 @@
+import {ISearchForm, ISearchForms} from "@comunica/actor-rdf-metadata-extract-hydra-controls";
+import {IActionRdfDereferencePaged, IActorRdfDereferencePagedOutput} from "@comunica/bus-rdf-dereference-paged";
+import {ActorRdfResolveQuadPatternSource, IActionRdfResolveQuadPattern,
+  IActorRdfResolveQuadPatternOutput, ILazyQuadSource} from "@comunica/bus-rdf-resolve-quad-pattern";
+import {ActionContext, Actor, IActorArgs, IActorTest, Mediator} from "@comunica/core";
+import * as RDF from "rdf-js";
+import {termToString} from "rdf-string";
+import {MediatedQuadSource} from "./MediatedQuadSource";
+
+/**
+ * A comunica tree RDF Resolve Quad Pattern Actor.
+ */
+export class ActorRdfResolveQuadPatternTree extends ActorRdfResolveQuadPatternSource
```

Comment:
Same comment here. This class is identical to the qpf rdf resolver. (And also the same issue with the mediated quad source). Having duplicated code creates an unmaintainable code base so this will need to change.